src/main/kotlin/uk/gov/dluhc/printapi/jobs/FinalRetentionPeriodDataRemovalJob.kt

```diff
@@ -14,6 +14,6 @@ class FinalRetentionPeriodDataRemovalJob(
     @Scheduled(cron = "\${jobs.remove-vca-final-retention-period-data.cron}")
     @SchedulerLock(name = "\${jobs.remove-vca-final-retention-period-data.name}")
     fun removeVoterCardFinalRetentionPeriodData() {
```

Comment:
Is it ok to have voterCard in the function name `removeVoterCardFinalRetentionPeriodData` ?

Reply:
Yes because I'm planning on adding separate jobs to remove temp certs and AEDs. This is so that their timings can vary if necessary, but also because they live in separate database tables, so it will just make it easier.